docs/www/data-migration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Migrating data to Redpanda
+order: 2
+---
+
+# Migrating data to Redpanda
+
+One of the more elegant aspects of Redpanda is that it is compatible with the [Kafka API and ecosystem](/docs/www/faq.md).
+So when you want to migrate data from Kafka or replicate data between Redpanda clusters,
+the MirrorMaker 2 tool bundled in the [Kafka download package](https://kafka.apache.org/downloads) is a natural solution.
+
+In this article, we'll go through the process of configuring MirrorMaker to replicate data from one Redpanda cluster to another.
+In short, MirrorMaker reads a configuration file that specifies the connection details for the Redpanda clusters, as well as other options.
+After you start MirrorMaker, the data migration continues according to the configuration at launch time until you shutdown the MirrorMaker process.
+
+For all of the details on MirrorMaker and its options, check out the [Kafka documentation](https://kafka.apache.org/documentation/#georeplication).
+
+## Prerequisites
+
+To set up the replication, we'll need:
+
+- 2 Redpanda clusters - You can set up these clusters in any deployment you like, including [Kubernetes](/docs/www/quick-start-kubernetes.md), [Docker](/docs/www/quick-start-docker.md), [Linux](/docs/www/quick-start-linux.md), or [MacOS](/docs/www/quick-start-macos.md).
+
+- MirrorMaker host - You install MirrorMaker on a separate system or on one of the Redpanda clusters, as long as the IP addresses and ports on each cluster are accessible from the MirrorMaker host.
+You must install the [Java Runtime Engine (JRE)](https://docs.oracle.com/javase/10/install/toc.htm) on the MirrorMaker host.
+
+## Installing MirrorMaker
+
+MirrorMaker 2 is run by a shell script that is part of the Kafka download package.
+To install MirrorMaker 2 on the machine that you want to run the replication between the clusters:
+
+1. Download the latest [Kafka download package](https://kafka.apache.org/downloads).
+
+    For example:
+
+    ```
+    curl -O https://dlcdn.apache.org/kafka/3.0.0/kafka_2.13-3.0.0.tgz
+    ```
+
+2. Extract the files from the archive:
+
+    ```
+    tar -xvf kafka_2.13-3.0.0.tgz
+    ```
+
+## Create the MirrorMaker 2 config files
+
+MirrorMaker uses configuration files to get the connection details for the clusters.
+You can find the MirrorMaker script and configuration files in the expanded Kafka directory.
+
+```
+.
+└── kafka_2.13-3.0.0
+    └── bin
+        └── connect-mirror-maker.sh
+        └── connect-mirror-maker.properties
```

Comment:
connect-mirror-maker.properties is under the `config` directory, not the `bin` directory